hvplot/utilities.py

```diff
@@ -24,3 +25,38 @@ def show(obj, title=None, port=0, **kwargs):
         raise ValueError('%s type object not recognized and cannot be shown.' %
                          type(obj).__name__)
     _pn.pane.HoloViews(obj).show(title, port, **kwargs)
+
+
+class hvplot_extension(_hv.extension):
```

Comment:
Hmm, how would you use this together with the import based extension loading mechanism? Does it end up embedding the JS/CSS twice?